app/views/sign_up_sheet/review_bids_others_work.html.erb

```diff
@@ -1,113 +1,103 @@
-<% session[:return_to] = request.url %>
-<% if flash[:notice] != nil and flash[:notice].strip != "" %>
-       <div class="flash_error">
-       <%= flash[:notice] %>
-       </div>
+<% def flash_message %>
```

Comment:
Could you add a comment that explains what this view shows?  It's kind of hard to figure out from reading the code.